rules/mocha.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  plugins: [
+    'mocha',
+  ],
+  rules: {
+    'mocha/no-exclusive-tests': 'error',
+    'mocha/no-skipped-tests': 'off',
+    'mocha/no-pending-tests': 'error',
```

Comment:
I see. You might (especially when things are still WIP) do sth. like this though:

```js
describe('new feature', () => {
  it('does thing a');

  it('does thing b');

  it('does thing c')
});
```

and fill those tests in later. I actually do that sometimes.